Services/GEV/Desktop/classes/class.gevMyTrainingsAdminTableGUI.php

```diff
@@ -185,52 +210,25 @@ protected function addActionMenuItems($current_selection_list, $a_set) {
 	}
 
 	protected function getActionMenuItems($a_set) {
-		//Prepare links
-		$this->gCtrl->setParameterByClass("gevMemberListDeliveryGUI", "ref_id", $a_set["crs_ref_id"]);
-		$memberlist_link = $this->gCtrl->getLinkTargetByClass("gevMemberListDeliveryGUI", "trainer");
-		$signature_list_link = $this->gCtrl->getLinkTargetByClass("gevMemberListDeliveryGUI", "download_signature_list");
-		$schedule_list_link = $this->gCtrl->getLinkTargetByClass("gevMemberListDeliveryGUI", "download_crs_schedule");
-		$csn_list_link = $this->gCtrl->getLinkTargetByClass("gevMemberListDeliveryGUI", "csn");
-		$this->gCtrl->clearParametersByClass("gevMemberListDeliveryGUI");
-		
-		$this->gCtrl->setParameter($this->parent_obj, "crsrefid", $a_set['crs_ref_id']);
-		$this->gCtrl->setParameter($this->parent_obj, "crs_id", $a_set['obj_id']);
-		$setstatus_link = $this->gCtrl->getLinkTarget($this->parent_obj, "listStatus");
-		$overnights_link = $this->gCtrl->getLinkTarget($this->parent_obj, "showOvernights");
-		$bookings_link = $this->gCtrl->getLinkTarget($this->parent_obj, "viewBookings");
-		$this->gCtrl->clearParameters($this->parent_obj);
-
-		$this->gCtrl->setParameterByClass("ilObjCourseGUI", "ref_id", $a_set["crs_ref_id"]);
-		$cancel_training_link = $this->gCtrl->getLinkTargetByClass("ilObjCourseGUI","confirmTrainingCancellation");
-		$this->gCtrl->clearParametersByClass("ilObjCourseGUI");
-
-		$this->gCtrl->setParameterByClass("gevMaillogGUI", "obj_id", $a_set["obj_id"]);
-		$maillog = $this->gCtrl->getLinkTargetByClass("gevMaillogGUI", "showMaillog");
-		$this->gCtrl->clearParametersByClass("gevMaillogGUI");
-
-		$this->gCtrl->setParameterByClass("ilObjCourseGUI", "ref_id", $a_set["crs_ref_id"]);
-		$edit_settings_link = $this->gCtrl->getLinkTargetByClass("ilObjCourseGUI", "edit");
-		$this->gCtrl->clearParametersByClass("ilObjCourseGUI");
-
 		//prepare crs utils
 		$crs_utils = gevCourseUtils::getInstance($a_set["obj_id"]);
 
```

Comment:
Da ich die Links auch an anderen Stellen in der fillRow Funktion benötige, finde ich das ich diese auch dort erstellen muss.